ui/util/remark-emote.js

```diff
@@ -1,4 +1,4 @@
-import { EMOTES_24px as EMOTES } from 'constants/emotes';
+import { EMOTES_72px as EMOTES } from 'constants/emotes';
 import visit from 'unist-util-visit';
 
 const EMOTE_NODE_TYPE = 'emote';
```

Comment:
Not a blocker, but there seems to be an error in the exchange rate calculation.
Also, the sentence feels inverted when in LBC mode.  Perhaps "Amount of <whatever entered> LBC in USD is lower than the price of $1" would flow better?

![image](https://user-images.githubusercontent.com/64950861/140521410-fe749703-1453-490b-8e68-8807633fab53.png)

Reply:
👍🏻  great correction